projects/aca-content/src/lib/components/search/search-results/search-results.component.scss

```diff
@@ -3,6 +3,13 @@
 $adf-chip-background: #efefef;
 $contrast-gray: #646569;
 
+.adf-toolbar-search-results {
+  position: fixed;
```

Comment:
this is probably the most lame way of positioning... putting flex everywhere, and then making fixed position like it is 1995 now